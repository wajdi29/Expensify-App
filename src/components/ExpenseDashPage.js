import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";
import ExpensesSummary from './ExpensesSummary'
import LoginPage from './LoginPage'
const ExpenseDashPage = () => (
  <div>
    <ExpensesSummary />
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);
export default ExpenseDashPage;
