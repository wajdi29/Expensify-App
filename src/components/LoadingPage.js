import React from 'react'
const LoadingPage = () => (
    <div className="loader">
        <img src="/img/loader.gif" className="loader__img" />
    </div>

)


export default LoadingPage