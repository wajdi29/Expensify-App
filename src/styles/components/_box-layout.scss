.box-layout {
  background: url("/img/bg.jpg");
  height: 100vh;
  width: 100vw;
}
