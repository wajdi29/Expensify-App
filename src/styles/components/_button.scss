.button {
  background: $blue;
  color: white;
  border: none;
  font-weight: 300;
  font-size: $font-size-large;
  padding: $s-size;
  display: inline-block;
  text-decoration: none;
  line-height: 1;
}
.button--link {
  background: none;
}
