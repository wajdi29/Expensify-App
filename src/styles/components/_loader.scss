.loader {
  align-items: center;
  height: 100vh;
  width: 100vw;
  display: flex;
  justify-content: center;
}

.loader__img {
  height: 6rem;
  width: 6rem;
}
