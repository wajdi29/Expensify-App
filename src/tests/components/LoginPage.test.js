import React from 'react'
import { shallow } from 'enzyme'
import { LoginPage } from '../../components/LoginPage'

test('should render Login page properly ', () => {
    const wrapper = shallow(<LoginPage />)
    expect(wrapper).toMatchSnapshot()
})