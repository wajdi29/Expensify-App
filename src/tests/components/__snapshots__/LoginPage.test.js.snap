// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Login page properly  1`] = `
<div>
  <button>
    Login
  </button>
</div>
`;
